README.md

```diff
@@ -216,7 +216,7 @@ com.example.myapplication.HasNamedCompanion2
 
 
 <details>
-<summary>Example of named companion rules for R8 full mode</summary>
+<summary>Example of named companion rules for R8 full-mode</summary>
```

Comment:
'Full mode' is mentioned in Google documentation without a dash:
https://developer.android.com/build/shrink-code#full-mode
https://r8.googlesource.com/r8/+/refs/heads/master/compatibility-faq.md#r8-full-mode

I prefer being consistent with them.